test/mixedapplyiter.jl

```diff
@@ -33,35 +27,35 @@ function mixed_metasumsq3(f, args...)
 	return res
 end
 
-function make_byref(out, fn, args...)
+function mixed_make_byref(out, fn, args...)
 	out[] = fn(args...)
 	nothing
 end
 
-function tupapprox(a, b)
-	if a isa Tuple && b isa Tuple
-		if length(a) != length(b)
-			return false
-		end
-		for (aa, bb) in zip(a, b)
-			if !tupapprox(aa, bb)
-				return false
-			end
-		end
-		return true
+function mixed_tupapprox(a, b)
+    if a isa Tuple && b isa Tuple
+	if length(a) != length(b)
+	    return false
 	end
-	if a isa Array && b isa Array
-		if size(a) != size(b)
-			return false
-		end
-		for i in length(a)
-			if !tupapprox(a[i], b[i])
-				return false
-			end
-		end
-		return true
+	for (aa, bb) in zip(a, b)
+	    if !mixed_tupapprox(aa, bb)
```

Comment:
so github here things the formatting is now messed up?

Reply:
There were still tabs 😱 It's [now fixed](https://github.com/EnzymeAD/Enzyme.jl/pull/2597#issuecomment-3315284183)